### ISSUE TITLE
Emoji picker: Can't reach skin tone selection via keyboard

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#728]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 2.1.1 (A) Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard).

Not everyone can use a mouse, other pointing device, or touchscreen. To make things usable for everyone, it's important for interactive elements to be usable via keyboard alone.

The skin tone selection part of the emoji picker is completely excluded from the keyboard focus order.
![Emoji picker with skin tone selector highlighted. ](https://github.com/user-attachments/assets/500d2838-5d6b-4ccf-8234-23f293cda1f9)

**To Reproduce**
Attempt to reach the skin tone selection part of the emoji picker using only keyboard navigation.

**Expected behavior**
Everything that's accessible by mouse should also be accessible by keyboard. Using standard semantic HTML elements instead of building custom ones out of `<div>` and `<span>` elements will usually achieve this.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
